vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -199,47 +200,73 @@ public Retrofit getRetrofit() {
         return mRetrofit;
     }
 
+    /**
+     * @return the current VimeoAccount, or null if it hasn't been set yet
+     */
+    @Nullable
     public VimeoAccount getVimeoAccount() {
-        if (mVimeoAccount == null) {
-            throw new AssertionError("Account should never be null");
-        }
+        return VimeoAccount.copy(mVimeoAccount);
+    }
 
-        return mVimeoAccount;
+    /**
+     * @return true if the current {@link VimeoAccount} is not null and has an auth token
+     */
+    public boolean isAuthenticated() {
+        return mVimeoAccount != null && mVimeoAccount.isAuthenticated();
     }
 
+    /**
+     * Sets the current {@link VimeoAccount} on this client. If the provided account is null and
+     * an access token exists on the {@link Configuration}, then a new account will be created
+     * using that access token.
+     *
+     * @param vimeoAccount the account to set on this client
+     */
     public void setVimeoAccount(@Nullable VimeoAccount vimeoAccount) {
         if (vimeoAccount == null) {
```

Comment:
I can clean this up for authenticating a config access token, but I think we should be able to null out the account somehow. Do you propose a different way of clearing out the account rather than allowing null to be passed in?